### ISSUE TITLE
In Topics page, keyboard navigation skips the content cards completely (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9190]
## Observed behavior
In Topics page, keyboard navigation skips the content cards completely and only focuses on the info icon.
![2022-03-14 15 07 16](https://user-images.githubusercontent.com/13563002/158269228-5d281c3c-dafd-4611-a210-edf090604b7f.gif)

## Errors and logs
None

## Expected behavior
Cards should also have focus. First focusable element should be the first available content card (typically a channel) - see below:
<img width="1487" alt="Screen Shot 2022-03-14 at 3 11 55 PM" src="https://user-images.githubusercontent.com/13563002/158269595-c026b919-a63e-4297-99aa-fb1af3e29391.png">

## User-facing consequences
User can't easily access content (though when the info panel does open, user can access the resource there)

## Steps to reproduce
1. Sign in as learner
2. Go to the library
3. Start to navigate library with keyboard. Notice that the first focusable element is the first info icon (skipping anything that is a clickable content card)

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Possible mac settings issues 

![card-navigation-firefox.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/150d724d-e8e0-4ab8-ab79-f6593c53d85a)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Confirmed - with the correct settings enabled the tab navigation works correctly on Safari, Firefox and Chrome on Mac - the cards can be tabbed through and the first focusable element is the first available content card.
